src/mono_camera.cpp

```diff
@@ -162,4 +179,37 @@ void MonoCamera::updateCameraInfo(const avt_vimba_camera::AvtVimbaCameraConfig&
   info_man_->setCameraInfo(ci);
 }
 
+void MonoCamera::publish_status()
+{
+
+ if (flag==1)
```

Comment:
Actually, instead of setting a flag variable, you can also set the status in the ROS message directly to avoid duplication.